### ISSUE TITLE
SD card booting do not work in my Lenovo 100e gen3 (zork) 

### ISSUE THREAD

**Issue by MarconesJr** [ading2210/shimboot#94]
1. Try to boot using the sd card port (i am using a 64gb sandisk ultra sd card)

**Expected behavior**
It will show that "factory instalation is aborted". 

**Screenshots / Photos**
![WhatsApp Image 2024-09-26 at 5 19 39 AM](https://github.com/user-attachments/assets/77b769da-5d68-443f-8e78-3286dfc1e9a5)

**Target Chrome OS Device:**
 - Zork chromebook
 - Vilboz device
 - Shimboot version: 1.2.1 
 - Using a prebuilt image

**Additional context**
if you use that same sd card with a USB adaptor it works fine.

**Comment by Tulip-0333** [comment]
Your SD card slot must be wacky. That error message happens when you unplug a USB or SD whilst it's booting. Have you maybe tried with a different person's computer? See if it's the model as a whole.

**Comment by Tulip-0333** [comment]
Also maybe try with a different SD card

**Comment by ading2210** [comment]
I don't believe this is fixable. But could you try switching to vt2 to show the error logs when this ocurrs? Press ctr+alt+f2 (or try all the keys in the function row)

**Comment by RowdyTJ** [comment]
Had the same issue with the same model Chromebook. Same error if you use the built in microsd reader. But it works fine if you just use a usb reader.

**Comment by RowdyTJ** [comment]
Flashed a stock image and still got the same issue in vt2

![Image](https://github.com/user-attachments/assets/a3e088cd-79b9-40c4-8144-44488bb4f3da)

**Comment by RowdyTJ** [comment]
> i forgot to say that I'm in window 11 education and my power button is on the side of the laptop and I've tried getting into recovery mode by pressing it
If you can boot windows, then you most likely have full UEFI ROM firmware and recovery mode does not exist. You won’t be able to use this project, just use normal Linux.

**Comment by jacobleclair** [comment]
doesn't make a difference. you're not using a chromebook, you're using windows. shimboot won't work.

what are you trying to achieve by using linux? you can probably get the same results using other methods. it's absurdly easy to get admin on any windows computer that you have physical access to. lmk and i can help you out.

this isn't an insult, i'm genuinely trying to warn you - if you weren't able to read that this project is for chromebooks... i don't think you want to run linux.

i'm also interested to hear, how did you stumble across this project? did you look up something like "play games on school computer" and you found this?

**Comment by jacobleclair** [comment]
i don't know much about either of those applications, but shimboot definitely isn't the way to bypass them. here's your easy options:

1.install a fresh copy of windows
2. install linux
3. bypass these systems

if your school doesn't lock stuff down too much (you have access to uefi settings, you can boot from external media, etc.), options 1-3 should be feasible. i don't know much about windows, but you might just be able to factory reset it to remove any school software. installing linux onto the laptop (or booting it from external media) may or may not be a good option, as it would be pretty easy to get caught. since windows looks the same out of the box, it would be easier to get away with linux. however, the best option is probably bypassing the systems, allowing for the most control and least chance of detection.

**⚠️ the steps below have a good chance of causing damage, be careful. ⚠️**

the steps below probably won't damage the computer itself, but it might get you suspended or something.

## reinstalling windows

this is one option. you probably want to use [microsoft's stuff](https://www.microsoft.com/en-us/software-download/windows11). an installation medium (the iso that you can download) might also be a good idea. if you take the installation assistant route, you might need admin, see the `bypass these systems` section.

## install linux

if you want linux, grap an iso of whatever distro you want, and install it. not much to say, as the installation process varies depending on distro.

## bypass the systems

this is probably the best route imo. windows' security is terrible, and it's very easy to get admin rights. i won't provide a step-by-step guide, but the way that i usually do it is turning the computer off, then turning it on, and while it's booting, turning it off again. then turn it back on, and turn it off when it's booting, and then turn it back on, then turn if off when it's booting, and then turn it back on one more time. it should do some kind of disk scan, and then it will put you in the recovery menu. you can open a command prompt, and from there move `utilman.exe` to `utilman.exe.bkp`, and copy `cmd.exe` to `utilman.exe.` finally, reboot and press the accessibility button on the login screen. it should open a command prompt. use the `net` command to create a local user and set it as an administrator. then, move `utilman.exe.bkp` back to `utilman.exe`. finally, reboot, and you should have an administrator account set up. **again, this isn't a step-by-step guide, it just gives an overview of what you will have to do.**

good luck! :)
